components/script/dom/readablestream.rs

```diff
@@ -59,6 +62,36 @@ pub struct ReadableStream {
 }
 
 impl ReadableStream {
+    #[allow(non_snake_case)]
+    // https://streams.spec.whatwg.org/#rs-constructor
+    pub fn Constructor(
+        cx: SafeJSContext,
+        _global: &GlobalScope,
+        _proto: Option<SafeHandleObject>,
+        underlying_source: Option<*mut JSObject>,
```

Comment:
But this is the signature required by the codegen.
And I think the spec also says it will convert to `UnderlyingSource` in step 2.
https://streams.spec.whatwg.org/#rs-constructor

Reply:
Right, ok.